src/Servers/Connections.Abstractions/src/Features/IUnidirectionalStreamFeature.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Connections.Features
+{
+    public interface IUnidirectionalStreamFeature
```

Comment:
How bad would it be to detect unidirectional via checking the output pipe being null or not?

Reply:
Terrible 😁. At least have a bool or enum property.